extensions/mssql/src/objectManagement/commands.ts

```diff
@@ -106,15 +107,19 @@ async function handleObjectPropertiesDialogCommand(context: azdata.ObjectExplore
 		return;
 	}
 	try {
-		const parentUrn = await getParentUrn(context);
+		const parentUrn = context.nodeInfo ? await getParentUrn(context) : undefined;
+		const objectType = context.nodeInfo && context.nodeInfo.nodeType === 'Database' ? ObjectManagement.NodeType.DatabaseProperties : context.nodeInfo.nodeType as ObjectManagement.NodeType;
+		const objectName = context.nodeInfo ? context.nodeInfo.label : objectManagementLoc.PropertiesHeader;
+		const objectUrn = context.nodeInfo ? context.nodeInfo!.metadata!.urn : undefined;
```

Comment:
Databases should have a URN too, so this shouldn't be necessary. Are you seeing an instance where it's not?